tslint.json

```diff
@@ -1,8 +1,12 @@
 {
   "extends": "tslint:latest",
   "rules": {
+    "ban-types": false,
     "interface-name": false,
     "interface-over-type-literal": false,
+    "max-classes-per-file": false,
+    "no-submodule-imports": false,
+    "no-console": [false],
```

Comment:
Added a few exceptions there. Regarding no-console it's not strictly necessary, it's just for quality of life